src/v2/cookbook/client-side-storage.md

```diff
@@ -95,39 +95,39 @@ const app = new Vue({
+Igual que antes, `mounted` es utilizado para cargar los datos almacenados cuando existen. Sin embargo esta vez los datos solo son guardados cuando hacemos clic en el botón. También se podrían efectuar validaciones antes guardar los datos. También se puede almacenar información sobre cuándo fueron guardados los datos. Con esa metadata el método `mounted` podría decidir si guardar o no los datos nuevamente. Puede probar esta nueva versión a continuación.
 
-<p data-height="265" data-theme-id="0" data-slug-hash="rdOjLN" data-default-tab="js,result" data-user="cfjedimaster" data-embed-version="2" data-pen-title="testing localstorage 2" class="codepen">See the Pen <a href="https://codepen.io/cfjedimaster/pen/rdOjLN/">testing localstorage 2</a> by Raymond Camden (<a href="https://codepen.io/cfjedimaster">@cfjedimaster</a>) on <a href="https://codepen.io">CodePen</a>.</p>
+<p data-height="265" data-theme-id="0" data-slug-hash="rdOjLN" data-default-tab="js,result" data-user="cfjedimaster" data-embed-version="2" data-pen-title="testing localstorage 2" class="codepen">Vea el Pen <a href="https://codepen.io/cfjedimaster/pen/rdOjLN/">probando localstorage 2</a> por Raymond Camden (<a href="https://codepen.io/cfjedimaster">@cfjedimaster</a>) en <a href="https://codepen.io">CodePen</a>.</p>
 <script async src="https://static.codepen.io/assets/embed/ei.js"></script>
 
-## Working with Complex Values
+## Trabajando con valores complejos
 
-As mentioned above, Local Storage only works with simple values. To store more complex values, like objects or arrays, you must serialize and deserialize the values with JSON. Here is a more advanced example that persists an array of cats (the best kind of array possible).
+Como fue mencionado anteriormente, Local Storage solamente funciona con valores simples. Para almacenar valores complejos como objetos o arreglos usted debe serializar y deserializar los valores con JSON. Aquí puede encontrar un ejemplo avanzado que almacena un arreglo de gatos (el mejor tipo de arreglos).
 
 ``` html
 <div id="app">
-  <h2>Cats</h2>
+  <h2>Gatos</h2>
   <div v-for="(cat, n) in cats">
     <p>
       <span class="cat">{{ cat }}</span>
-      <button @click="removeCat(n)">Remove</button>
+      <button @click="removeCat(n)">Eliminar</button>
     </p>
   </div>
 
   <p>
     <input v-model="newCat">
-    <button @click="addCat">Add Cat</button>
+    <button @click="addCat">Agregar gato</button>
   </p>
 </div>
 ```
 
-This "app" consists of a simple list on top (with a button to remove a cat) and a small form at the bottom to add a new cat. Now let's look at the JavaScript.
+Está "aplicación" consiste en una simple lista con un botón para eliminar gatos y un formulario para agregarlos en la parte inferior. Veamos el JavaScript.

```

Comment:
`Está "aplicación"`
Please, change "Está" to "Ésta"

Reply:
Actually, the E should not have accent, because the word is a "llana" that ends in a vowel

Reply:
You are right @pepesenaris. 
 @fedekau Can you change from `Está "aplicación` to `Esta "aplicación"`? 

Thanks to both

Reply:
Thank you both, great catch 👏